src/material/sort/sort-header.scss

```diff
@@ -32,6 +32,13 @@ $mat-sort-header-arrow-hint-opacity: 0.38;
   font: inherit;
   color: currentColor;
 
+  // Usually we could rely on the arrow showing up to be focus indication, but if a header is
+  // active, the arrow will always be shown so the user has no way of telling the difference.
+  [mat-sort-header].cdk-keyboard-focused &,
+  [mat-sort-header].cdk-program-focused & {
+    outline: solid 1px currentColor;
```

Comment:
Note: not sure whether an outline is too jarring. We could also go for a text underline if we want something more subtle.

Reply:
I wouldn't say it's _jarring_, but I do think it's a bit too tight around the text. I tried `border-bottom` locally and that seems less intrusive, so we could go with that for now and try to get some feedback from the material design team.